projects/observability/src/shared/dashboard/data/graphql/table/entity/entity-table-data-source.model.ts

```diff
@@ -80,12 +110,113 @@ export class EntityTableDataSourceModel extends TableDataSourceModel {
     response: EntitiesResponse,
     request: TableDataRequest<SpecificationBackedTableColumnDef>
   ): TableDataResponse<TableRow> {
+    let results: Entity<string>[] = response.results;
+
+    /**
+     * This block only runs when the table is client side filtered and rendered.
```

Comment:
Ambiguous terminology. Could be mistaken that otherwise the table is [server side rendered][1]. We're only changing the filtering I think. The rendering method is still CSR either way, right?

[1]: https://web.dev/rendering-on-the-web/#client-side-rendering-csr

Reply:
Yeah, makes sense, would call it `filtered`.